dataplane/api-server/src/lib.rs

```diff
@@ -42,23 +44,37 @@ pub async fn start(
     let healthchecks = tokio::spawn(async move {
         let (_, health_service) = tonic_health::server::health_reporter();
         let mut server_builder = Server::builder();
-        server_builder
+
+        let port = port + 1;
+        let addr = SocketAddrV4::new(addr, port);
+        let server = server_builder
             .add_service(health_service)
-            .serve(SocketAddrV4::new(addr, port + 1).into())
-            .await
-            .unwrap();
+            .serve(addr.into());
+
+        debug!("gRPC Health Checking service listens on port {}", port);
+        server.await.map_err(|e| {
+            error!("Failed serve gRPC Health Checking service, err: {:?}", e);
+            e
+        }).unwrap();
```

Comment:
Since we're going to unwrap and panic anyway (meaning, we'll log the error, then we'll get a panic logged right after), I think we might as well just do a nice clear `.expect()` here with our error message.